misk-hibernate-testing/src/main/kotlin/misk/hibernate/HibernateTestingModule.kt

```diff
@@ -50,7 +51,7 @@ class HibernateTestingModule(
         qualifier = qualifier,
         config = configProvider.get(),
         transacterProvider = transacterProvider,
-        checks = checks,
+        checks = if (useVitessChecks) checks else null,
```

Comment:
checks is a confusing name for something that's vitess specific. Could we rename this to the full `vitessScaleSafetyChecks`?

Reply:
done